ci/requirements-py3.11.yml

```
@@ -0,0 +1,32 @@
+name: test_env
+channels:
+    - defaults
+    - conda-forge
+dependencies:
+    - coveralls
+    # - cython
+    # - ephem
+    - h5py
+    # - netcdf4  # ???
+    # - nose
+    # - numba  # ???
+    - numpy >= 1.16.0
+    - pandas >= 0.25.0
+    - pip
+    - pytest
+    - pytest-cov
+    - pytest-mock
+    - requests-mock
+    - pytest-timeout
+    - pytest-rerunfailures
+    - pytest-remotedata
+    - python=3.11
+    - pytz
+    - requests
+    - scipy >= 1.2.0
+    # - shapely  # ??? pvfactors dependency
+    # - siphon  # conda-forge
+    - statsmodels
+    - pip:
+        # - nrel-pysam>=2.0  # install error on windows
+        - pvfactors==1.4.1
```

```suggestion
        - pvfactors==1.5.2
```

The two failing conda jobs are failing because they're using shapely 2.0 but pvfactors isn't compatible with shapely 2.0. 
[pvfactors v1.5.2](https://sunpower.github.io/pvfactors/whatsnew.html#v1-5-2-date-xx-2021) added an upper limit to its shapely dependency to address this until compatibility is added, so using 1.5.2 here should pull in shapely 1.8.5 instead of 2.0.0 and get the jobs passing.  I think pinning the shapely version (instead of updating the pvfactors version) would also work, but might as well test on the latest pvfactors anyway.